src/config.ts

```diff
@@ -4,59 +4,60 @@ import {
   IntegrationInstanceConfigFieldMap,
   IntegrationInstanceConfig,
 } from '@jupiterone/integration-sdk-core';
-import { createAPIClient } from './client';
+import { Client } from './client';
+import { ParsedServiceAccountKeyFile } from './utils/parseServiceAccountKeyFile';
 
-/**
- * A type describing the configuration fields required to execute the
- * integration for a specific account in the data provider.
- *
- * When executing the integration in a development environment, these values may
- * be provided in a `.env` file with environment variables. For example:
- *
- * - `CLIENT_ID=123` becomes `instance.config.clientId = '123'`
- * - `CLIENT_SECRET=abc` becomes `instance.config.clientSecret = 'abc'`
- *
- * Environment variables are NOT used when the integration is executing in a
- * managed environment. For example, in JupiterOne, users configure
- * `instance.config` in a UI.
- */
 export const instanceConfigFields: IntegrationInstanceConfigFieldMap = {
-  clientId: {
+  type: {
     type: 'string',
   },
-  clientSecret: {
+  project_id: {
+    type: 'string',
+  },
+  private_key_id: {
+    type: 'string',
+    mask: true,
+  },
+  private_key: {
     type: 'string',
     mask: true,
   },
+  client_email: {
+    type: 'string',
+    mask: true,
+  },
+  client_id: {
+    type: 'string',
+  },
+  auth_uri: {
+    type: 'string',
+  },
+  token_uri: {
+    type: 'string',
+  },
+  auth_provider_x509_cert_url: {
+    type: 'string',
+  },
+  client_x509_cert_url: {
+    type: 'string',
+  },
```

Comment:
This doesn't agree with how the config file is handled. You can look at the `further-tweaks` branch for reference.

Reply:
Might also be the prime culprit why `yarn start` doesn't run.